metricflow/test/snapshots/test_conversion_metrics_to_sql.py/SqlQueryPlan/BigQuery/test_conversion_metric_fill_nulls_with_0__plan0_optimized.sql

```diff
@@ -0,0 +1,84 @@
+-- Compute Metrics via Expressions
+SELECT
+  metric_time__day
+  , CAST(buys AS FLOAT64) / CAST(NULLIF(visits, 0) AS FLOAT64) AS visit_buy_conversion_rate_7days_fill_nulls_with_0
```

Comment:
Oh interesting. If we want to support recursively filling nulls with something we'll need to apply it on the metric, because conversion and ratio metrics will return NULL for any zero denominator.

This is the default behavior we want - NULL is the correct result - but users may override when they know, for example, that any zero denominator implies a 0% conversion rate for their analysis purposes.

Nothing to do about that now except maybe make a note of this in the test docstring.